### ISSUE TITLE
What is the meaning of `softmax` in attention_module.py?

### ISSUE THREAD

**Issue by theodoruszq** [tengshaofeng/ResidualAttentionNetwork-pytorch#30]
Hi, I am confused about the term `softmax_blocks`. The term in the paper should be `soft mask blocks`? I check the ResidualBlock class which does not exist normalization layers.
